server/backend/testspg/conftest.py

```diff
@@ -266,3 +265,56 @@ def auth_headers(test_rsa_key):
     token = jwt_utils.create_jwt_token(test_rsa_key)
     headers = jwt_utils.headers(token)
     return headers
+
+
+@pytest.fixture(scope="function")
+def setup_new_user(db_pg_session: Session):
+    """
+    New user setup for testing.
+    The fixture returns a function to be called with new user created based on
+        user_type, user_name and optionally if need to add cognito_user_id.
+    """
+
+    def _setup_new_user(
+        user_type: UserType, user_name, user_guid, cognito_user_id: Optional[str] = None
+    ) -> FamUser:
+        new_user_create = FamUserSchema(
+            **{
+                "user_type_code": user_type,
+                "user_name": user_name,
+                "user_guid": user_guid,
+                "create_user": TEST_CREATOR,
+                "first_name": "Fist",
+                "last_name": "Last",
+                "email": "test@test.com"
+            }
+        )
+
+        fam_user = crud_user.create_user(new_user_create, db_pg_session)
+        if cognito_user_id is not None:
+            # SqlAlchemy is a bit strange, need to use `.query()` to do the
+            # update() and query() again in order to get correct updated entity
+            # from session.
+            db_pg_session.query(FamUser).filter(
+                FamUser.user_id == fam_user.user_id
+            ).update({FamUser.cognito_user_id: cognito_user_id})
+
+            fam_user = (
+                db_pg_session.query(FamUser)
+                .filter(FamUser.user_id == fam_user.user_id)
+                .one()
+            )
+
+        return fam_user
+
+    return _setup_new_user
+
+
+@pytest.fixture(scope="function", autouse=True)
+def mock_forest_client_integration_service():
```

Comment:
Existing fixture migrate from "test_forest_client_validator" test to conftest.py to avoid test failure due to mock and share with other tests.